awspub/configmodels.py

```diff
@@ -94,6 +95,34 @@ class ConfigImageSSMParameterModel(BaseModel):
     )
 
 
+class ConfigSNSNotificationProtocol(str, Enum):
+    DEFAULT = "default"
+    EMAIL = "email"
+
+
+class ConfigImageSNSNotificationModel(BaseModel):
+    """
+    Image/AMI SNS Notification specific configuration to notify subscribers about new images availability
+    """
+
+    model_config = ConfigDict(extra="forbid", use_enum_values=True)
+
+    subject: str = Field(description="The subject of SNS Notification", min_length=1, max_length=99)
```

Comment:
with `use_enum_values` do all Fields have to be an enum? its not clear from the documentation and haven't looked at tests yet to confirm we use this model in them.

Reply:
No, it doesn’t need to be an enum. It’s used to handle values instead of raw enums. However, since we don’t need to print out these protocol keys anywhere, this flag can be dropped.